pingpong/rdm_cntr_pingpong.c

```diff
@@ -327,16 +327,10 @@ static int init_fabric(void)
 	char *node, *service;
 	int ret;
 
-	if (opts.dst_addr) {
-		ret = ft_getsrcaddr(opts.src_addr, opts.src_port, hints);
-		if (ret)
-			return ret;
-		node = opts.dst_addr;
-		service = opts.dst_port;
-	} else {
-		node = opts.src_addr;
-		service = opts.src_port;
-		flags = FI_SOURCE;
+	ret = ft_read_addr_opts(&node, &service, hints, &flags, &opts);
+	if (ret) {
+		FT_PRINTERR("ft_read_addr_opts", ret);
+		return ret;
```

Comment:
Please keep to 1 concept per patch.  This change looks unrelated to selecting the address.